zebra/redistribute.c

```diff
@@ -730,8 +730,12 @@ void zebra_import_table_rm_update()
 				continue;
 
 			rmap_name = zebra_get_import_table_route_map(afi, i);
-			if (!rmap_name)
-				return;
```

Comment:
Nice catch on the return here, definately should be a continue to look at other afi's.